.teamcity/settings.kts

```diff
@@ -388,11 +389,13 @@ object RunAllUnitTests : BuildType({
 				(cd apps/o2-blocks/ && yarn build --output-path="../../artifacts/o2-blocks")
 
 				# Build wpcom-block-editor
-				(cd apps/wpcom-block-editor/ &&  NODE_ENV=development yarn build --output-path="../../artifacts/wpcom-block-editor")
+				(cd apps/wpcom-block-editor/ && NODE_ENV=development yarn build)
+				zip -r ../../artifacts/wpcom-block-editor.zip apps/wpcom-block-editor/dist/
 
 				# Build notifications
 				(cd apps/notifications/ && yarn build --output-path="../../artifacts/notifications")
 			"""
+			dockerImage= "%docker_image_wpcom%"
```

Comment:
note: on the default image, `zip` is not available. I'm not sure what ramifications this has. Should we install .zip in the default image instead of this?